src/main/java/com/karankumar/booksapi/repository/BookRepository.java

```diff
@@ -27,5 +28,7 @@
 
     Book findBookByIsbn13(String isbn13);
   
+    Book findByGenre( String genre);
```

Comment:
There may be multiple books in a genre, so could you please change the return type to a `List<Book>`

Reply:
Also, we don't want to search by a `String`. Rather, we would like to pass in our enum, `BookGenre`.